### DISCUSSION TITLE
Readme

### DISCUSSION THREAD

**Discussion by Caaasperrr** [community/community#62609] [Repositories]
### Select Topic Area

Question

### Body

How do i add a fancy Readme.md file?

**Comment by FelixHennerich** [comment]
Hi, 
you have to create a repository with the exact same name as your Github name. Afterwords you can add a file named README.md where you can write your Readme text. 

Tipp: You can advance your with headlines by using # at the beginn of a line. 

I Hope it helps :)

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
